src/Stratis.Bitcoin.Features.BlockStore/Controllers/BlockStoreController.cs

```diff
@@ -264,6 +265,34 @@ public IActionResult GetVerboseAddressesBalancesData(string addresses)
             }
         }
 
+        /// <summary>Provides verbose balance data of the given addresses.</summary>
+        /// <param name="addresses">A comma delimited set of addresses that will be queried.</param>
+        /// <returns>A result object containing the balance for each requested address and if so, a message stating why the indexer is not queryable.</returns>
+        /// <response code="200">Returns balances for the requested addresses</response>
+        /// <response code="400">Unexpected exception occurred</response>
+        [Route(BlockStoreRouteEndPoint.GetVerboseAddressesBalances2)]
+        [HttpPost]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        public IActionResult GetVerboseAddressesBalancesData2([FromBody] string addresses)
```

Comment:
This is a nitpick, but can we name this `VerboseAddressesBalancesData` instead, with endpoint `verboseaddressesbalances`? It looks cleaner than appending the `2`